src/main/kotlin/com/hitc/hitweconomyplugin/main/utils/GameFileUtils.kt

```diff
@@ -1,41 +1,135 @@
 package com.hitc.hitweconomyplugin.main.utils
 
+import com.hitc.hitweconomyplugin.main.core.DataFile
+import com.hitc.hitweconomyplugin.main.core.EPlayer
+import com.hitc.hitweconomyplugin.main.core.PlayerData
 import com.hitc.hitweconomyplugin.main.core.Score
 import kotlinx.serialization.decodeFromString
 import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromJsonElement
+import kotlinx.serialization.json.encodeToJsonElement
+import org.bukkit.entity.Player
 import java.io.File
-import java.lang.Exception
+import java.nio.file.Files
 import java.util.*
+import java.util.zip.DataFormatException
+import kotlin.Exception
 
 object GameFileUtils {
 
-    fun loadScores(uuid : UUID) : MutableList<Score?> {
-        var scores : MutableList<Score?> = MutableList(0) { null }
-        val stringUUID = uuid.toString()
-        val f = File("./plugins/HitW/scores/$stringUUID.json")
-        if (!f.exists()) f.createNewFile()
+    fun loadScores(ePlayer: EPlayer,
+                   calendar: Calendar = Calendar.getInstance(TimeZone.getTimeZone("EST"))
+    ) : MutableList<Score?> {
+        var scores = MutableList<Score?>(0) { null }
```

Comment:
`(0) { null }` probably unnecessary maybe the Score? make it mandatory but I don't think so